package.json

```diff
@@ -8,6 +8,8 @@
     "server:html": "node ./src/http-servers​/html-server.js",
     "server:json": "node ./src/http-servers​/json-server.js",
     "server:echo": "node ./src/http-servers​/echo-server.js",
+    "swagger:edit": "swagger project edit",
```

Comment:
'swagger' is not recognized as an internal or external command.
And even when I install it, I get an error `Cannot GET /`.